R/ucsc.R

```diff
@@ -270,19 +270,28 @@ setMethod("ucscTableQuery", "character",
                    names = NULL, intersectTrack = NULL, check = TRUE, hubUrl = NULL,
                    genome = NULL, url = "http://genome.ucsc.edu/cgi-bin/") {
               stopifnot(isSingleString(x))
-              if (!isSingleString(table))
-                stop("'table' is a mandatory parameter and must be a single character vector")
               if (!is.null(intersectTrack))
                 stop("intersectTrack is no longer supported")
-              if (!is.null(track))
-                warning("track is meaningless now you only go by the table")
+              # if (!is.null(track))
+              #   warning("track is meaningless now you only go by the table")
```

Comment:
We will need to issue a deprecation warning so that packages know to start updating.